src/envoy/http/authn/http_filter_integration_test.cc

```diff
@@ -131,118 +144,57 @@ TEST_P(AuthenticationFilterIntegrationTest, OriginJwtRequiredHeaderNoJwtFail) {
 }
 
 TEST_P(AuthenticationFilterIntegrationTest, CheckValidJwtPassAuthentication) {
-  createTestServer(
-      "src/envoy/http/authn/testdata/envoy_origin_jwt_authn_only.conf",
-      {"http"});
+  config_helper_.addFilter(kAuthnFilterWithJwt);
+  config_helper_.addFilter(MakeHeaderToMetadataConfig());
```

Comment:
can we just real JWT filter?  It is integration test, should not use fake filter to reject fake data?

Reply:
Then I have to make this dependent on JWT filter. This integration test should be for this filter only, so we don't have to dig to much in other JWT filter change.

Btw, I am also working on adding another integration test, that set up the same filter chain we have for Istio (jwt + authn + mixer) which can cover the real end-to-end flow.